Description: bin/: Use 'require' or 'require_relative'
 As we're installing into '/usr/lib/ruby/vendor_ruby',
 'require_relative' as used by upstream doesn't work.

 A test which was added in the last upstream release made this change no
 longer work, thus we're catching a possible 'LoadError' now and require
 the file via a relative path.

 Upstream is aware of this patch and we're working on integrating it.
Author: Georg Faerber <georg@riseup.net>
Last-Update: 2017-10-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/bin/schleuder
+++ b/bin/schleuder
@@ -9,8 +9,11 @@
 
 begin
   require_relative '../lib/schleuder/cli'
-  Schleuder::Cli.start
+rescue LoadError
+  require 'schleuder/cli'
 rescue => exc
   $stderr.puts "Technical Error: #{exc}\n#{exc.backtrace.first}"
   exit 1
 end
+
+Schleuder::Cli.start
--- a/bin/schleuder-api-daemon
+++ b/bin/schleuder-api-daemon
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
 
-require_relative '../lib/schleuder-api-daemon'
-SchleuderApiDaemon.run!
\ No newline at end of file
+require 'schleuder-api-daemon'
+SchleuderApiDaemon.run!
